civihr_default_theme/scss/components/_ui-select.scss

```diff
@@ -74,35 +74,8 @@
 
 /*
 * TEMP fixes till ui-select styling is fixed
+* Dropdown ul opacity was set to 0
 */
-.civihr-ui-select {
```

Comment:
good questions, tried to to git bisect and changed to earlier version of civicrm, could not locate the cause of regression yet